docs/models/index.md

```diff
@@ -1,16 +1,67 @@
+
+2. **OpenEdx statements models**:
+    - [Enrollment](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#enrollment-events)
+    - [Navigational](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#course-navigation-events)
+    - [Open Reponse Assessment](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#open-response-assessment-events)
+    - [Peer instruction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#peer-instruction-events)
+    - [Problem interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#problem-interaction-events)
+    - [Textbook interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#textbook-interaction-events)
+    - [Video interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#video-interaction-events)
+
+## Statements validation
+
+In learning analytics, the validation of statements takes on significant importance. 
+These statements, originating from diverse sources, systems or applications, must align with specific standards such as [xAPI](https://xapi.com/) or [Caliper](https://www.imsglobal.org/activity/caliper), for the best known. 
+The validation process becomes essential in ensuring that these statements meet the required standards, facilitating data quality and reliability. 
+
+Ralph proposes a validation statements tool allowing you to automate the validation process in your production stack. 
+OpenEdx related events and xAPI statements are supported in the project. 
+
+Based on Pydantic modelisation for the set of statements supported by Ralph, the validation feature provides a streamlined, reliable, and efficient approach to validate large volumes of learning statements.
+
+!!! warning 
+
+    For now, validation is effective only with supported [learning statement models](#learning-statement-models) on Ralph. About xAPI statements, an [issue](https://github.com/openfun/ralph/issues/388) is open to extend validation to any xAPI statement.
+
+Check out tutorials to test the validation feature:
+
+- [`validate` with Ralph as a CLI](../tutorials/cli.md#validate-command)
+- [`validate` with Ralph as a library](../tutorials/library.md#validate-method)
+
+## Statements conversion
+
+The statements conversion feature offers a seamless transition between OpenEdx and xAPI statement formats, promoting interoperability and adaptability. 
+
+By converting OpenEdx statements to xAPI, you ensure that your learning data can be easily integrated into various e-learning ecosystems.
```

Comment:
```suggestion
```
In my opinion, the typical reader will not need to be convinced that conversion is a good idea (either they need it or they don't)